t/02-rakudo/15-gh_1202.t

```diff
@@ -0,0 +1,26 @@
+use v6;
+use Test;
+
+# GH#1202
+
+plan 2;
+
+my Bool $complete = False;
+
+await Promise.anyof(
+    start {
+        my $out = "";
+        # This was a bug with unpredictable output. Hopefully 50 repetiotions is enough to make sure everything works.
+        for 1..50 -> $rep {
+            my $proc = run $*EXECUTABLE, '-MTest', '-e', q<await (^5).map({start { print qqx{echo $_} } })>, :out;
```

Comment:
Always use `$*EXECUTABLE.absolute` with `run` or `shell`. Why are you using `-MTest`? Does that one-liner work on windows with its weird quoting rules?

Reply:
Ok. Test is just overlooked here, I was playing with `is` in the subprocess. Will remove.

Reply:
As to Windows - it seems to be working under appveyor. I'll have a chance testing on my win 7 VM tomorrow.

Reply:
I tested on windows and it does what is expected. However the very first time I got the follow error (which I haven't replicated since):
```
C:\Users\ugexe\source>perl6 -e "my $proc = run $*EXECUTABLE, '-MTest', '-e', q<await (^5).map({start { print qqx{echo $_} } })>, :out;"

C:\Users\ugexe\source>An operation first awaited:
  in block <unit> at -e line 1

Died with the exception:
    Failed to write bytes to filehandle: Invalid argument
      in block  at -e line 1
```

Reply:
On my VM it never failed. I merge and close #1202.